golos.publication/golos.publication.abi

```diff
@@ -192,6 +192,10 @@
         "name": "id",
         "type": "uint64"
       },
+      {
+        "name": "account",
+        "type": "name"
+      },
```

Comment:
We don't need to save `account` in this record. This information already saved in scope field.
When you create `message_table`, you pass `account.value` as second argument. So this table contains records only for this author.